controller/refugee_controller.py

```diff
@@ -1,42 +1,72 @@
 ###---- Manage Refugee Menu ----
 from datetime import date
-
+from enum import Enum
+from typing import Type
+from models.plan import Plan
 from models.camp import Camp
 from models.refugee import Refugee
+from controller.controller_error import ControllerError
+
+
+def list_medical_condition_types() -> Type[Enum]:
+    """
+    List all options for medical condition type.
+    :return:
+    """
+    return Refugee.MedicalCondition()
 
 
-def create_refugee(firstname: str,
+def create_refugee(user_id: str,
+                   firstname: str,
                    lastname: str,
                    camp: Camp,
                    num_of_family_member: int,
                    starting_date: date,
                    medical_condition_type: Refugee.MedicalCondition) -> Refugee:
     """
-    Yingbo, Michelle
-    A function used by admin and volunteer.
+    Function to create and save refugee.
```

Comment:
Maybe 'create and save refugee data' would be better to avoid ambiguity??

Reply:
Thanks for the suggestion! I have changed the comment to avoid ambiguity.